### PR TITLE
Add information about how to contribute

### PR DESCRIPTION
**Pull request by mishan**
@TysonAndre @monnand

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,21 @@ data. Make sure that the Redis server you use has persistence enabled - your
 redis.conf should have contents similar to the section `**PERSISTENCE**` of
 redis.conf in the example config files linked in http://redis.io/topics/config
 
+# Contributing #
+You're encouraged to contribute to the uniqush-push project. There are two ways you can contribute.
+
+# Issues #
```
Comment (TysonAndre) [inline README.md]: I think you use two (`##`) on each side for subheadings.

http://maruku.rubyforge.org/maruku.md http://maruku.rubyforge.org/maruku.html

File: README.md [hunk 2]
```diff
@@ -73,6 +73,21 @@ data. Make sure that the Redis server you use has persistence enabled - your
...
+
+If you encounter an issue while using uniqush-push, please report it at the project's issues tracker. Feature suggestions are also welcome.
```
Comment (TysonAndre) [inline README.md]: nit: `uniqush-push`

File: README.md [hunk 3]
```diff
@@ -73,6 +73,21 @@ data. Make sure that the Redis server you use has persistence enabled - your
...
+
+# Pull request #
+
+Code contributions to uniqush-push can be made using pull requests. To submit a pull request:
```
Comment (TysonAndre) [inline README.md]: nit: `uniqush-push`

File: README.md (hunk 2 above)
Comment (TysonAndre) [inline README.md]: could be a markdown link to issues list

**Comment by mishan** [comment]
@TysonAndre updated.

File: README.md [hunk 4]
```diff
@@ -65,15 +65,31 @@ about the latest progress with *Uniqush*, please check out [our
...
 data. Make sure that the Redis server you use has persistence enabled - your
 redis.conf should have contents similar to the section `**PERSISTENCE**` of
 redis.conf in the example config files linked in http://redis.io/topics/config
 
-# Related Links #
+## Contributing ##
+
+You're encouraged to contribute to the `uniqush-push` project. There are two ways you can contribute.
+
+## Issues ##
```
Comment (TysonAndre) [inline README.md]: Should be `<h3>` below `<h2`> ?

**Comment by mishan** [comment]
changed to h3

**Comment by mishan** [comment]
squashed commits

**Pull request merged**